src/main/resources/assets/minecolonies/lang/en_us.json

```
@@ -34,8 +34,11 @@
   "minecolonies.config.generatesupplyloot": "Supply loot",
   "minecolonies.config.generatesupplyloot.comment": "Whether to generate supply ship/camps in chests occasionally.",
 
-  "minecolonies.config.forceloadcolony": "Force Load Colony",
-  "minecolonies.config.forceloadcolony.comment": "If part of the colony is loaded by an owner/officer, the entire colony should be kept loaded.",
+  "minecolonies.config.forceloadcolony": "Chunk Load Colony",
+  "minecolonies.config.forceloadcolony.comment": "If part of the colony is loaded by an owner/officer, the colony should be kept loaded.",
+
+  "minecolonies.config.colonyLoadStrictness": "Colony Chunk Loading Strictness",
+  "minecolonies.config.colonyLoadStrictness.comment": "This controls how many chunks are loaded with the "Chunk Load Colony" option. The higher this value, the less chunks will be loaded. (The innermost chunks will be loaded first.) 1 = load all claimed chunks.",
```

```suggestion
  "minecolonies.config.colonyLoadStrictness.comment": "This controls how many chunks are loaded with the \"Chunk Load Colony\" option. The higher this value, the less chunks will be loaded. (The innermost chunks will be loaded first.) 1 = load all claimed chunks.",
```
sorry forgot backslashes